### PR TITLE
added styling

### PR DESCRIPTION
**Pull request by rijusougata13**

### CODE REVIEW

File: snowy-silence-43C/index.html [hunk 1]
```diff
@@ -22,14 +22,14 @@ <h1>{{ header.toLocaleUpperCase() }}</h1>
       </ul>
       <p>{{itemCount}} things.</p>
       <p v-if="items.length === 0"><i>Nothing is epic :(</i></p>
-      <h2>LONks</h2>
+      <h2>--LONks--</h2>
```
Comment (ICTman1076) [inline snowy-silence-43C/index.html]: Not sure what the `--` does - could you explain a little further?

File: snowy-silence-43C/index.html [hunk 2]
```diff
@@ -5,10 +5,10 @@
   <meta name="viewport" content="width=device-width">
   <title>Very epic vue.js</title>
   <script src="https://unpkg.com/vue"></script>
-  <style>.notEpic{background-color:#FF0000;} .virus{color:#FF00FF;} .notVirus{color:#00FF00;}</style>
+  <style>.notEpic{background-color:coral;} .virus{color:red;} .notVirus{color:green;}</style>
```
Comment (ICTman1076) [inline snowy-silence-43C/index.html:8]: Coral's a much better color, and using word-based colors makes sense. Nice work 👍

**Review by ICTman1076 — CHANGES_REQUESTED** [review]
Thanks for your PR! I'm a little confused by line 25, but other than that this is nice. Could you provide some explanation where asked please? Thanks :)

**Comment by rijusougata13** [comment]
"--lonks--" is  looking nice than "lonks".it doesn't matter actually.
but the colour "coral" matter in this case.

**Review by ICTman1076 — APPROVED** [review]
Ok, I've removed the double dash, but I'll keep everything else. I'm happy to merge this now. Thanks again :)

**Comment by rijusougata13** [comment]
ok

**Pull request merged**